collections/_release_4_5/entry-platforms-android-edge-to-edge-support.md

```diff
@@ -0,0 +1,19 @@
+---
+type: entry
+section: platforms
+subsection: android
+rank: 3
+importance: 4
+anchor: edge-to-edge-support
+title: Edge-to-edge support
+text: |
+  Usually, developers had two choices for displaying content on the screen: either their app leaves the top and bottom areas for the system status and navigation bars, or their app could go full-screen, claiming all the space without displaying these bars.
+
+  From now on, there’s a new export option. Android has a feature called ["edge-to-edge"](https://developer.android.com/develop/ui/views/layout/edge-to-edge) display. It offers the developers the ability to draw on the entire screen, as if it was fullscreen, but with system bar overlays.
+
+  This gives your game a more modern look that matches Android’s design style.
+contributors:
```

Comment:
@adamscott Can you also include @syntaxerror247 as contributor since he provided numerous bug fixes to get the feature stable.